packages/contracts-da-bridge/contracts/script/DeployDemo.s.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.7.6;
+
+import "forge-std/Script.sol";
+import {XAppConnectionManager} from "@nomad-xyz/contracts-core/contracts/XAppConnectionManager.sol";
+import {Replica} from "@nomad-xyz/contracts-core/contracts/Replica.sol";
+import {DABridgeRouter} from "../DABridgeRouter.sol";
+
+contract DeployDABridgeRouter is Script {
```

Comment:
@luketchang checkout https://book.getfoundry.sh/cheatcodes/parse-json

seems like a better ux (and less error prone) than env variables